ros_buildfarm/bloom_status.py

```diff
@@ -79,7 +103,11 @@ def _get_package_status(g, name, branch_info):
         return 'NO RELEASE'
     try:
         repo = g.get_repo(branch_info['org'] + '/' + name)
-        the_diff = repo.compare(branch_info['release'], branch_info['upstream'])
+        if 'last_release' in branch_info:
```

Comment:
Could this be `branch = branch_info.get('last_release', branch_info.get('release'))`?